UIforETW/Utility.cpp

```diff
@@ -431,108 +645,117 @@ bool Is64BitWindows()
 #else
 	// http://blogs.msdn.com/b/oldnewthing/archive/2005/02/01/364563.aspx
 	BOOL f64 = FALSE;
-	bool bIsWin64 = IsWow64Process(GetCurrentProcess(), &f64) && f64;
-	return bIsWin64;
+	return ::IsWow64Process(::GetCurrentProcess(), &f64) && f64;
 #endif
 }
 
 bool Is64BitBuild()
 {
+	//when MSVC gets constexpr, this is a perfect candidate.

```

Comment:
I'm not sure that constexpr would improve this code since we don't actually need to declare a variable. But, the ideal solution is probably #if defined(_WIN64), matching Is64BitWindows() above.